config/locales/ar.yml

```diff
@@ -0,0 +1,1039 @@
+en:

```

Comment:
You'll want to rename this to `ar:`

And then for the right-hand text on each line below, you'll want to translate the English text.